uaclient/exceptions.py

```diff
@@ -16,6 +16,22 @@ def __init__(self, msg: str) -> None:
         self.msg = msg
 
 
+class BetaServiceError(Exception):
+    """
+    An exception to be raised trying to interact with beta service
+    without the right parameters.
+
+    :param msg:
+        Takes a single parameter, which is the beta service error message that
+        should be emitted before exiting non-zero.
+    """
+
+    exit_code = 1
+
+    def __init__(self, msg: str) -> None:
+        self.msg = msg
+
+
```

Comment:
This should subclass from UserFacingError right, then you also don't have to add/redefine exit_code = 1 as a class attr here right?

```suggestion
class BetaServiceError(Exception):
    """
    An exception to be raised trying to interact with beta service
    without the right parameters.

    :param msg:
        Takes a single parameter, which is the beta service error message that
        should be emitted before exiting non-zero.
    """
    pass


```